python/pyspark/sql/readwriter.py

```diff
@@ -239,6 +239,9 @@ def json(self, path, schema=None, primitivesAsString=None, prefersDecimal=None,
                                           including tab and line feed characters) or not.
         :param lineSep: defines the line separator that should be used for parsing. If None is
                         set, it covers all ``\\r``, ``\\r\\n`` and ``\\n``.
+        :param samplingRatio: defines fraction of rows (when ``multiLine`` is ``false``) or fraction
```

Comment:
@MaxGekk, can we just don't say it's for files when `multiLine` is enabled? I think JSON makes a sample for each record (or an array of JSONs) regardless of record per file or multiple records in a file at high level.

Reply:
Will it be ok if I write something like:
```
samplingRation defines fraction of input json objects used for schema inferring
``` 
Please, give me your variants if it doesn't work for you.

Reply:
+1 (json -> JSON)